external/thorntail-mp-tck/playlist.xml

```diff
@@ -27,7 +27,7 @@
 		</groups>
 	</test>
 	<test>
-		<testCaseName>thorntail_microprofile_tck</testCaseName>
+		<testCaseName>thorntail-mp-tck_test</testCaseName>
```

Comment:
thorntail test will be removed https://github.com/adoptium/aqa-tests/issues/3182 (so it won't matter that this is using '-')